app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/ScheduleActivity.java

```diff
@@ -1,15 +1,59 @@
 package ch.epfl.sweng.eventmanager.ui.eventShowcase;
 
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
 
 import ch.epfl.sweng.eventmanager.R;
+import ch.epfl.sweng.eventmanager.repository.data.Concert;
+import ch.epfl.sweng.eventmanager.ui.eventShowcase.scheduleUtils.TimeLineAdapter;
+import ch.epfl.sweng.eventmanager.ui.eventShowcase.scheduleUtils.TimeLineModel;
 
 public class ScheduleActivity extends AppCompatActivity {
 
+    private RecyclerView recyclerView;
+    private List<TimeLineModel> dataList = new ArrayList<>();
+    private TimeLineAdapter timeLineAdapter;
+    private List<Concert> concertsList;
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_schedule);
+
+        recyclerView = findViewById(R.id.recyclerView);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false));
+        recyclerView.setHasFixedSize(true);
+
+        //
+        // Get the concert List for the event into concertsList
+        //
+        concertsList = new LinkedList<>();
```

Comment:
You added a concert list in `Event`, use that one!

Reply:
We did not decide during this Sprint how to pass the ConcertList to the ScheduleActivity. This is not the final code.

Reply:
Okay, but then why did you put Concert objects in the `Event` class?

Reply:
Removed.